example-nextjs/pages/ts.tsx

```diff
@@ -92,7 +104,7 @@ console.log(a + b);`)
               <Editor
                 value={output?.join('\n') ?? ''}
                 height="10rem"
-                defaultLanguage="typescript"
+                defaultLanguage="markdown"
```

Comment:
Apologies for introducing a change irrelevant to the topic of this PR, but this improves the output viewing experience. :D